index.coffee

```diff
@@ -41,9 +42,13 @@ fillwidth = ($list, targetHeight, done, apply, gutterSize) ->
       img.width = img.width * (targetHeight / img.height)
       img.height = targetHeight
 
+    # Return the callback if images already fit, and resizeUp is false
+    imagesWidth = imgsWidth()
```

Comment:
This is probably cheap enough to not have to store the result in a variable, but 👍  works for me.